yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
```

Comment:
Do we commit this file?

Reply:
Thinking that we dont need to commit yarn.lock file? cc @KMKoushik

Reply:
We need to commit this file!